core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -102,6 +104,16 @@ public Object visitReplaceCommand(InvocationContext ctx, ReplaceCommand command)
       return handleTxWriteCommand(ctx, command, command.getKey());
    }
 
+   @Override
+   public Object visitComputeCommand(InvocationContext ctx, ComputeCommand command) throws Throwable {
+      return handleTxWriteCommand(ctx, command, command.getKey());
```

Comment:
After giving it more thought, I've realized this is a non-issue, but please add this comment in case I'll forget:
```
Contrary to functional commands, compute() needs to return the new value returned by the remapping function. Since we can assume that old and new values are comparable in size, fetching old value into context is acceptable and it is more efficient in case that we execute more subsequent modifications than sending the return value.
```